subsystem/cluster_test.go

```diff
@@ -5290,18 +5312,60 @@ var _ = Describe("Verify install-config manifest", func() {
 
 		// Completing cluster installation
 		completeInstallationAndVerify(ctx, agentBMClient, clusterID, true)
-	})
+	}
 
 	AfterEach(func() {
 		deregisterResources()
 		clearDB()
 	})
 
-	It("Validate install-config.yaml content retrieved from V2DownloadClusterFiles", func() {
-		validateInstallConfig(getInstallConfigFromFile())
-	})
+	Context("Validate install-config content", func() {
+		tests := []struct {
```

Comment:
you can use https://onsi.github.io/ginkgo/#table-specs

Reply:
Good idea. Changed to DescribeTable.